src/components/combat/Combat.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '/src/styles/common.scss';
+@import url('https://fonts.cdnfonts.com/css/proxima-nova-alt-lt-2');
+
+                
+$darkBlue: #1f2343;
+
+@mixin mobile {
+  @media (max-width: 786px) {
+    @content;
+    flex-direction: column;
+  }
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  font-family: 'Proxima Nova Alt Lt'; 
+  padding: 50px 0px; 
```

Comment:
there isnt left-right padding when I open the link compare to your picture

Reply:
50px Padding added both left and right